### DISCUSSION TITLE
Custom notifications and multiple "instances" of the same agent

### DISCUSSION THREAD

**Discussion by lukezona** [sct/overseerr#2150] [Feature Requests]
Hi, I'd love to have more options to customize the notifications like in Tautulli:
- use the same agent multiple times, that would be useful to use different channels on Telegram filtering the notifications (for example receive the requests on the Admin channel and the availabilities in the public one)
- customize messages, I'm Italian so it'd also be useful to localize them manually, but it would allow me to set only the variables I want to see (for example hide the "Requested by" and the link for Overseer), in Tautulli the message field can be written like "New {type} available: {title}"

I know the last bit may be too much (it adds quite a bit of complexity), but would be awesome to at least be able to choose which fields to show (hiding the "Requested by", "Status" and the link for a specific channel would be awesome)

Hope it can be added sooner or later.

**Comment by TheCatLady** [comment]
Localization of notifications is planned.

As for more complex notification logic, the webhook agent is always an option.

**Reply by lukezona** [reply]
Thanks for the suggestion about the webhook, I didn't think about it.

I'm having issues though: from Postman I can send a JSON payload and get a response, the same payload with the Overseerr webhook returns a 403 on the bot server (I had this error with Postman if I didn't set the content to JSON)

Should I open a different issue for this?

**Comment by zoravar08** [comment]
any update on this
